mon-pix/tests/unit/routes/authentication/login-saml_test.js

```diff
@@ -0,0 +1,31 @@
+import { setupTest } from 'ember-mocha';
+import { describe, it } from 'mocha';
+import Service from '@ember/service';
+import sinon from 'sinon';
+import PixWindow from 'mon-pix/utils/pix-window';
+
+describe('Unit | Routes | authentication | login-saml', function () {
+  setupTest();
+
+  describe('#beforeModel', function () {
+    context('when a token is set as an hash of an url', function () {
+      it('should authenticate the user', async function () {
+        // given
+        const token = 'token';
```

Comment:
**remarque**
mettre un autre texte que "token" à minima ? (symbol or not ?)

Reply:
Malheureusement utiliser Symbol n'est pas possible à cause de la ligne 21 `.returns('#token')`.